modules/csg/csg_shape.cpp

```diff
@@ -2224,11 +2046,11 @@ void CSGPolygon3D::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::PACKED_VECTOR2_ARRAY, "polygon"), "set_polygon", "get_polygon");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "mode", PROPERTY_HINT_ENUM, "Depth,Spin,Path"), "set_mode", "get_mode");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "depth", PROPERTY_HINT_EXP_RANGE, "0.001,1000.0,0.001,or_greater"), "set_depth", "get_depth");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "depth", PROPERTY_HINT_EXP_RANGE, "0.01,100.0,0.01,or_greater"), "set_depth", "get_depth");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "spin_degrees", PROPERTY_HINT_RANGE, "1,360,0.1"), "set_spin_degrees", "get_spin_degrees");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "spin_sides", PROPERTY_HINT_RANGE, "3,64,1"), "set_spin_sides", "get_spin_sides");
 	ADD_PROPERTY(PropertyInfo(Variant::NODE_PATH, "path_node", PROPERTY_HINT_NODE_PATH_VALID_TYPES, "Path3D"), "set_path_node", "get_path_node");
-	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "path_interval", PROPERTY_HINT_EXP_RANGE, "0.001,1000.0,0.001,or_greater"), "set_path_interval", "get_path_interval");
```

Comment:
Why did you change the range of path_interval from "0.001,1000.0,0.001,or_greater" to "0.1,1.0,0.1" ?
For exemple In my slide of my proposal I am using around "0.145".
at least should be "0.01,1.0,0.01" but in same time that limit us a lot in possibility.
Maybe can be "0.005,1.0,0.001" to save us a lot of process with "0.001" of path_interval.

Reply:
@stebulba I assume you agree that it should never need to be greater than 1. Therefore, the question is about the minimum and the step size.

What is your use case for having a smaller minimum and/or smaller step sizes? Since `path_interval` simply sub-divides each extrusion without adding fidelity. If a user wants to increase the number of extrusions, I would expect the preferred approach to be to increase the number of points on the curve and leave `path_interval` at 1.

It's also worth pointing out that this is just a limitation in the editor. A minimum of 0.1 and a step size of 0.1 seemed reasonable for making adjustments in the editor. However, the user can always set the path interval to any value between (0:1] in code using `set_path_interval()` or the property `path_interval`.

Reply:
I understand.
- I used path_interval 0.152 and 360 points to do the 360 degrees.
- For approximately the same result path_interval 1 and 60 points to do the 360 degrees.
And the render I a lot better. This point is ok.

But I have a bug. When the extrudes follow the curve points, is rotating around it. 
What expected:
![copy_paste1](https://user-images.githubusercontent.com/26193340/124755923-f3984e00-def9-11eb-9c56-b8541db3966f.png)
What I got:
![copy_paste2](https://user-images.githubusercontent.com/26193340/124755988-ff841000-def9-11eb-89e6-7c2159dbf390.png)
I racing car will get trouble keep on track.